.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: 'Bug Report: (fill in)'
```

Comment:
In order to help to quickly identify the issue type and easily sort them, could you prepend the title with the type of the issue?

Something like:

```suggestion
title: '[Bug] <title>'
```

Could you do the same for the other templates as well?